inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/mq/MessageQueueZoneProducer.java

```diff
@@ -107,6 +107,7 @@ public void run() {
      */
     public void reload() {
         try {
+            LOG.info("start reload mq clusters");
```

Comment:
This log seems has not enough info, will this log output too much?

Reply:
no, low frequency